RELEASE

```diff
@@ -1,2 +1,2 @@
-v0.0.4
-- Adding user feedback about code validity on downloading original video
+v0.1.0
```

Comment:
package.json version is inconsistent with 0.1.0

  "version": "0.0.4",

Reply:
That was fixed in #97f9750